Dockerfile

```diff
@@ -181,10 +181,9 @@ RUN JAVA_ARCH=$( if [ "$TARGETARCH" = "arm64" ]; then echo -n "aarch64"; else ec
     ln -s /usr/local/jdk-* /usr/local/java
 
 # Install DB migration tool
-RUN GO_MIGRATE_ARCH=$( if [ "$TARGETARCH" = "arm64" ]; then echo -n "arm64"; else echo -n "amd64"; fi ) && \
-    curl -sLo - https://github.com/${GO_MIGRATE_FORK}/migrate/releases/download/v$GO_MIGRATE_VERSION/migrate.${TARGETOS}-${GO_MIGRATE_ARCH}.tar.gz | \
-    tar -xzf - -O > /usr/local/bin/migrate && \
-    chmod +x /usr/local/bin/migrate
```

Comment:
The comment above about bringing us back in-line with OSS migrate tool is correct. This is intended as they now have more CVE fixes.

I think we may still need the `chmod +x /usr/local/bin/migrate` after untar?

Reply:
ok - I can submit a PR to OSS for that

Reply:
https://github.com/LF-Decentralized-Trust-labs/paladin/pull/649